src/main/java/org/usfirst/frc/team4099/robot/Constants.kt

```diff
@@ -21,6 +35,18 @@ class Constants {
         val RIGHT_MASTER_ID = 4
         val RIGHT_SLAVE_1_ID = 5
         val RIGHT_SLAVE_2_ID = 0
+
+        val HIGH_GEAR_MAX_SETPOINT = 17.0 * 12.0 //17 fps
```

Comment:
Is this how the max setpoints work in Phoenix? I thought they got rid of that 12V scaling factor